### DISCUSSION TITLE
前端图片压缩 有推荐的包吗

### DISCUSSION THREAD

**Discussion by hihuangwei** [ant-design/ant-design#41942] [General]
前端图片压缩 有推荐的包吗

**Comment by afc163** [comment]
不建议在前端做图片压缩。

**Reply by hihuangwei** [reply]
前端不压缩的话，手机拍摄的图片上传太大了，有的时候都是十几兆，有什么解决方案

**Reply by afc163** [reply]
可以搜一下 https://zh.lmgtfy.app/?q=%E5%89%8D%E7%AB%AF%E5%81%9A%E5%9B%BE%E7%89%87%E5%8E%8B%E7%BC%A9
